src/net/net.zig

```diff
@@ -268,6 +400,15 @@ pub const Ipv6Addr = struct {
     pub fn isMulticast(self: *const Self) bool {
         return self.octets[0] == 255;
     }
+
+    pub fn to_hex(self: *const Self, alloc: std.mem.Allocator) ![]const u8 {
+        const ipv6_hex = try std.fmt.allocPrint(
+            alloc,
+            "{s}:{s}:{s}:{s}",
+            .{ std.fmt.fmtSliceHexLower(self.octets[0..4]), std.fmt.fmtSliceHexLower(self.octets[4..8]), std.fmt.fmtSliceHexLower(self.octets[8..12]), std.fmt.fmtSliceHexLower(self.octets[12..16]) },
```

Comment:
add `,` at the of `octets[12..16])` and re-run `zig fmt` so we dont have one large line